Makefile

```diff
@@ -569,7 +569,8 @@ HIGHEND_SRC = \
             telemetry/hott.c \
             telemetry/smartport.c \
             telemetry/ltm.c \
-            telemetry/mavlink.c
+            telemetry/mavlink.c \
+			telemetry/esc_telemetry.c \
```

Comment:
![image](https://cloud.githubusercontent.com/assets/334779/20169061/ad6a55f6-a725-11e6-8964-29c231949100.png)
Something is fishy here, well, i will just set it to Soft instead of Auto.